### ISSUE TITLE
Performer.io broken?

### ISSUE THREAD

**Issue by djmeijer** [blakewest/performer#2]
I want to take a look at this project but performer.io doesn't seem to work. Tested it on macOS with Safari, Chrome and Firefox. Piano rendering is visible, when I select a song and try to play it won't work. Also tried to upload a midi file, doesn't work either.

**Comment by Ten0** [comment]
It doesn't work indeed.

**Comment by blakewest** [comment]
@djmeijer @Ten0 yeah, playing songs stopped working for some reason. I didn't change any of the code, and I do recall it stopping working one day. Which makes me think it may have been some browser related changes. Do you see a particular error message?

**Comment by blakewest** [comment]
I did really like this project, and it was pretty sweet to see it work, so I'd be down to try and put in a little time to get this up and running again.

**Comment by Ten0** [comment]
That's cool! I hope to use the generated fingerings to learn pieces, maybe writing something to interface Performer with Synthesia.
I saw something trying to read on a null or undefined variable in my browser console at some point (on performer.io and not on my local install). Browser update is indeed another possible problem, though I didn't see any browser depreciation/method call block message.

**Comment by blakewest** [comment]
@djmeijer I fixed this! Sorry it's quite late. This was a really old side project for me. But it's christmas break time, and I finally said, let's see what's up! Turns out it was a fairly easy fix. But I tested this on a fresh install, and it appears to work for me. There's now a shrinkwrap file, and I fixed it so if you change anything in dev, those changes will actually take effect. 😄 . I haven't updated the heroku app yet. That appears to be totally broken. But I may try to fix that as well. In the mean time, it should work locally!
